sn_networking/src/record_store.rs

```diff
@@ -279,19 +279,11 @@ impl NodeRecordStore {
     /// Calculate the cost to store data for our current store state
     #[allow(clippy::mutable_key_type)]
     pub(crate) fn store_cost(&self) -> NanoTokens {
-        let relevant_records_len = if let Some(distance_range) = self.distance_range {
-            let record_keys: HashSet<_> = self.records.keys().cloned().collect();
-            self.get_records_within_distance_range(&record_keys, distance_range)
-        } else {
-            warn!("No distance range set on record store. Returning MAX_RECORDS_COUNT for relevant records in store cost calculation.");
-            MAX_RECORDS_COUNT
-        };
-
-        let cost =
-            calculate_cost_for_relevant_records(relevant_records_len, self.received_payment_count);
+        let stored_records = self.records.len();
+        let cost = calculate_cost_for_relevant_records(stored_records, self.received_payment_count);
```

Comment:
Hmm, this is no longer for relevant records just all records, no? we'd need to change the name... but does that not skew pricing for longer running nodes? (or nodes that have experienced more churn?)

Reply:
caculate `relevance` is costly, as for each key need to `sort peers to check self is in` .
hence changed to this much simplified version.

I think for longer running nodes, this will give them the benefit of reduce their claimed charge,
and let them have a higher chance of winning.
Which is somehow a reward for their long stay with the network ?

Reply:
what is `costly` here, `13ms/313ms`?